### PR TITLE
[Doc] add documents for bitwise functions

### PR DESCRIPTION
**Pull request by zbtzbtzbt**
# Proposed changes

add documents for bitwise functions

## Problem Summary:

Describe the overview of changes.

## Checklist(Required)

1. Does it affect the original behavior: (No)
2. Has unit tests been added: (No Need)
3. Has document been added or modified: (No)
4. Does it need to update dependencies: (No)
5. Are there any changes that cannot be rolled back: (No)

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: docs/en/sql-reference/sql-functions/bitwise-functions/bitand.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
+---
```
Comment (EmmyMiao87) [inline docs/en/sql-reference/sql-functions/bitwise-functions/bitand.md:1]: Please add document into sidebar also ~

**Review by hf200012 — DISMISSED** [review]
Cancel

**Review by hf200012 — COMMENTED** [review]
Cannel

**Review by EmmyMiao87 — CHANGES_REQUESTED** [review]
Please add document into sidebar also ~

File: docs/en/sql-reference/sql-functions/bitwise-functions/bitand.md (hunk 1 above)
Comment (zbtzbtzbt) [reply docs/en/sql-reference/sql-functions/bitwise-functions/bitand.md:1]: done

**Review by morningman — APPROVED** [review]
LGTM

**Review by EmmyMiao87 — APPROVED** [review]
LGTM

**Pull request merged**